packages/docs-app/src/examples/core-examples/sliderExample.tsx

```diff
@@ -80,11 +80,11 @@ export class SliderExample extends BaseExample<ISliderExampleState> {
         return (value: number) => this.setState({ [key]: value });
     }
 
-    private renderLabel1(val: number) {
+    private labelRenderer1(val: number) {
```

Comment:
since it's a function, we should stick to `renderLabel`

Reply:
Yep